alt/asyncapi/test/commands/context.test.ts

```diff
@@ -0,0 +1,17 @@
+import {expect, test} from '@oclif/test'
```

Comment:
Please forget about all the test files. They're generated and I didn't update them.